### PR TITLE
REF: Make member inplace rename available for identifier owners

### PR DESCRIPTION
**Pull request by mchernyavsky**
See [comment](https://intellij-support.jetbrains.com/hc/en-us/community/posts/360000014084/comments/360000019204) for explanation.

### CODE REVIEW

File: src/main/kotlin/org/rust/ide/refactoring/RsRefactoringSupportProvider.kt [hunk 1]
```diff
@@ -12,9 +12,14 @@ import org.rust.ide.refactoring.extractFunction.RsExtractFunctionHandler
...
 import org.rust.lang.core.psi.RsPatBinding
+import org.rust.lang.core.psi.ext.RsReferenceElement
 
 class RsRefactoringSupportProvider : RefactoringSupportProvider() {
-    override fun isInplaceRenameAvailable(element: PsiElement, context: PsiElement?): Boolean = element is RsPatBinding
+    override fun isInplaceRenameAvailable(element: PsiElement, context: PsiElement?): Boolean =
+        element is RsPatBinding
+
+    override fun isMemberInplaceRenameAvailable(element: PsiElement, context: PsiElement?): Boolean =
+        element is RsReferenceElement
```
Comment (Undin) [inline src/main/kotlin/org/rust/ide/refactoring/RsRefactoringSupportProvider.kt]: In what cases does it should work?
Looks like element is always not `RsReferenceElement` even we call rename on reference element

Comment (mchernyavsky) [reply src/main/kotlin/org/rust/ide/refactoring/RsRefactoringSupportProvider.kt]: That's a really good point!

Comment (mchernyavsky) [reply src/main/kotlin/org/rust/ide/refactoring/RsRefactoringSupportProvider.kt]: Fixed.

**Pull request merged**